src/lib/Libwin/win_remote_shell.c

```diff
@@ -252,10 +257,10 @@ create_std_pipes(STARTUPINFO* psi, char *pipename_append, int is_interactive)
  * @retval	!0 - error number, while trying to connect named pipe
  *
  */
-int
+DWORD
```

Comment:
Why are we changing this ?

Reply:
This function returns the error from GetLastError function which has a return type of DWORD.